packages/components/src/StatusIndicator/StatusIndicator.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./StatusIndicator.css";
+
+export type StatusType =
+  | "success"
+  | "warning"
+  | "critical"
+  | "inactive"
+  | "informative";
+
+interface StatusIndicatorProps {
+  readonly status: StatusType;
+}
+
+export function StatusIndicator({ status }: StatusIndicatorProps) {
+  return (
+    <div
+      style={{ backgroundColor: `var(--color-${status}` }}
```

Comment:
What's our stance on style attributes being used instead of classes? Just wondering as I've typically seen the styles given a name based on the status.

Reply:
yeah we normally create a CSS file for it and then reference that + use the .d.ts to generate the types we need.

https://github.com/GetJobber/atlantis/blob/a5704355cdc4ffef2919c12a4dbd028c675df909/packages/components/src/ProgressBar/ProgressBarStepped.tsx#L10

https://github.com/GetJobber/atlantis/blob/a5704355cdc4ffef2919c12a4dbd028c675df909/packages/components/src/ProgressBar/Sizes.css#L1-L11

Mostly so we can take advantage of CSS goodness when we need to.

With that said, you mostly just moved this to its own component so i don't mind it.